internal/services/files.go

```diff
@@ -952,9 +952,19 @@ func (f *Files) ScrapeAndStoreImage(rawURL string) (uuid.UUID, error) {
 		return uuid.Nil, nil
 	}
 
-	resImage, err := http.Get(rawURL)
+	req, err := GetGetRequestForUrl(rawURL)
 	if err != nil {
-		return uuid.Nil, err
+		return uuid.Nil, nil
```

Comment:
This should be `return uuid.Nil, err` because there was an error fetching the resource.